tiledb/sm/fragment/fragment_info.cc

```diff
@@ -516,16 +538,16 @@ Status FragmentInfo::get_mbr_var_size(
         Status_FragmentInfoError("Cannot get MBR var size; End "
                                  "size argument cannot be null"));
 
-  if (fid >= fragments_.size())
+  if (fid >= fragment_num())
     return LOG_STATUS(Status_FragmentInfoError(
         "Cannot get MBR var size; Invalid fragment index"));
 
-  if (!fragments_[fid].sparse())
+  if (!single_fragment_info_vec_[fid].sparse())
     return LOG_STATUS(
         Status_FragmentInfoError("Cannot get MBR; Fragment is not sparse"));
 
-  auto meta = fragments_[fid].meta();
-  RETURN_NOT_OK(meta->load_rtree(*array_->encryption_key()));
+  auto meta = single_fragment_info_vec_[fid].meta();
+  RETURN_NOT_OK(meta->load_rtree(enc_key_));
   const auto& mbrs = meta->mbrs();
 
   if (mid >= mbrs.size())
```

Comment:
Nit: did, mid, fid in this file make it hard to read for me. We use d, f in other parts of the code, then maybe to a full mbr_idx?

Reply:
These are from previous iterations, let's address in a separate PR holistically.